### PR TITLE
Add tools category 'Other' with K8s related tools

### LINKED ISSUE #2295: Tools other than CLI, GUI and Proxy
There are some (probably many) tools that don't fit in the categories we have on the Tools page. I'm thinking about these:

* https://github.com/oliver006/redis_exporter Promethius metrics exporter
* https://github.com/IBM/operator-for-redis-cluster Kubernetes operator for Redis Cluster

Is it possible to add an "Other" category on the Tools page? Can I simply create a new directory under [tools/](https://github.com/redis/redis-doc/tree/master/tools)?

@itamarhaber @nermiller WDYT?

### PR DESCRIPTION
**Pull request by zuiderkwast**
Fixes #2295

### CODE REVIEW

File: tools/other/github.com/oliver006/redis_exporter.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "name": "Prometheus Redis Metrics Exporter",
+    "description": "Prometheus Exporter for Redis Metrics"
```
Comment (bjosv) [inline tools/other/github.com/oliver006/redis_exporter.json:3]:
```suggestion
    "description": "Prometheus exporter for Redis metrics."
```

**Comment by zuiderkwast** [comment]
@oliver006, @4n4nd, @ese we're adding your projects to the Redis docs site on the page https://redis.io/resources/tools/. Anything you want to add or change in description, etc?

**Review by bjosv — APPROVED** [review]
LGTM, just a nit.

**Comment by 4n4nd** [comment]
@cin is the maintainer of IBM/operator-for-redis-cluster. I am just a contributor.

**Review by itamarhaber — APPROVED** [review]

**Comment by zuiderkwast** [comment]
CI didn't catch the illegal JSON? Could be useful to add...

**Comment by oliver006** [comment]
Thanks for letting me know - looks good!

**Pull request merged**